app/assets/javascripts/controllers/ems_common/ems_common_form_controller.js

```diff
@@ -206,7 +206,7 @@ ManageIQ.angular.app.controller('emsCommonFormController', ['$http', '$scope', '
       ($scope.emsCommonModel.emstype) &&
       ($scope.emsCommonModel.hawkular_hostname != '' && $scope.emsCommonModel.hawkular_api_port)) {
       return true;
-    }else if($scope.emsCommonModel.emstype == "gce" && $scope.emsCommonModel.project != '' &&
+    } else if($scope.emsCommonModel.emstype == "gce" && $scope.emsCommonModel.project != '' &&
```

Comment:
a. This specific `else if` was introduced in the _multi endpoint for containers providers_ PR and I wanted to fix that.
b. All over the file it's `if(` without space, i do not want to change all the ifs in the file.